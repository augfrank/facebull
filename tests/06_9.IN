M0   C0   C1  2716
M1   C0   C2  2839
M2   C0   C3  2860
M3   C0   C4  2182
M4   C0   C5  2548
M5   C1   C0  2722
M6   C1   C2  2378
M7   C1   C3  2610
M8   C1   C4  2865
M9   C1   C5  2266
M10   C2   C0  2269
M11   C2   C1  2665
M12   C2   C3  2591
M13   C2   C4  2309
M14   C2   C5  2838
M15   C3   C0  2506
M16   C3   C1  2336
M17   C3   C2  2435
M18   C3   C4  2275
M19   C3   C5  2420
M20   C4   C0  2154
M21   C4   C1  2815
M22   C4   C2  2568
M23   C4   C3  2852
M24   C4   C5  2678
M25   C5   C0  2315
M26   C5   C1  2950
M27   C5   C2  2497
M28   C5   C3  2236
M29   C5   C4  2313
