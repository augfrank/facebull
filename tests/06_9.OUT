13961
4 6 13 16 20 28
